ansible/roles/test/files/ptftests/fdb_test.py

```diff
@@ -36,16 +31,11 @@ def setUp(self):
         self.dataplane = ptf.dataplane_instance
         self.fdb = fdb.Fdb(self.test_params['fdb_info'])
         self.vlan_ip = ip_address(unicode(self.test_params['vlan_ip']))
+        self.dummy_mac_prefix = self.test_params["dummy_mac_prefix"]
+        self.dummy_mac_number = int(self.test_params["dummy_mac_number"])
+        self.dummy_mac_table = {}
 
         self.setUpFdb()
-
-        self.log("Start arp_responder")
-        self.shell(["supervisorctl", "start", "arp_responder"])
```

Comment:
>arp_responder [](start = 47, length = 13)

why the supervisorctl is removed?

Reply:
This code is to start arp_responder on the PTF container. Arp_responder is to reply to ARP request. It was not used in this FDB testing at all. So I removed irrelevant code.